bindings/dotnetcore/src/dotnetcore.cpp

```diff
@@ -34,13 +34,13 @@ typedef void(*PGatewayDestroyDelegate)(unsigned int moduleIdManaged);
 
 typedef void(*PGatewayStartDelegate)(unsigned int moduleIdManaged);
 
-static PGatewayCreateDelegate GatewayCreateDelegate;
```

Comment:
Yeah, I will keep an eye on it. As of now, I don't see this as a problem. Also, the guideline is not that the user calls it. This is going to be called by our Gateway Wrapper (Microsoft.Azure.IoT.Devices.Gateway.CreateFromJson...). And it's the next work item.